src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -101,7 +104,7 @@ def get(self, request: Request, organization: Organization) -> Response:
                 )
 
         createdBy = serializer.validated_data.get("createdBy", "me")
-        sort = SORT_MAP[serializer.validated_data.get("sort", "-visited")]
+        sorts = [SORT_MAP[sort] for sort in serializer.validated_data.get("sort", ["-visited"])]
```

Comment:
do we need this second argument if we have default=["-visited"] above

Reply:
Yeah good catch - deleted